statistics/histogram.go

```diff
@@ -2280,3 +2274,44 @@ func MergePartitionHist2GlobalHist(sc *stmtctx.StatementContext, hists []*Histog
 	}
 	return globalHist, nil
 }
+
+// ColLoadedStatus indicates the status of column statistics
+type ColLoadedStatus struct {
+	wasLoaded         bool
+	cmsEvicting       bool
+	topnEvicting      bool
+	histogramEvicting bool
```

Comment:
It is strange that "was" and "evicting" appears in status...
What about keeping loaded in case of any incompatible possibility. And one enum to represent the sub-loaded status like: cmsEvicted & allEvicted. 
We don't need to separate booleans because we evict in only one order and we don't have combined status like cms exists but hist&topn evicted.

Reply:
updated, use single status instead